plugins/ckan_operators/resource_operator.py

```diff
@@ -9,53 +9,186 @@
 
 class GetOrCreateResourceOperator(BaseOperator):
     """
-    Returns CKAN resource object, or creates it if it does not exist 
+    Returns CKAN resource object, or creates it if it does not exist
```

Comment:
All these changes below are either:
- making the GetOrCreateResourceOperator receive task info or actual values as inputs
- documenting that inline